test/integration/test_conferences.py

```diff
@@ -0,0 +1,488 @@
+        """
+        Set up for our tests by creating the CallsApi and ConferencesApi instances
+        for testing as well as the unauthorized and forbidden credentials for the 4xx tests.
+        """
+        configuration = bandwidth.Configuration(
+            username = BW_USERNAME,
+            password = BW_PASSWORD
+        )
+        api_client = bandwidth.ApiClient(configuration)
+
+        # Two Valid API Clients
+        self.calls_api_instance = calls_api.CallsApi(api_client)
+        self.conference_api_instance = conferences_api.ConferencesApi(api_client)
+
+        # Unauthorized API Client
+
+        unauthorizedConfiguration = bandwidth.Configuration(
+             username='bad_username',
+             password='bad_password'
+        )
+        unauthorized_api_client = bandwidth.ApiClient(unauthorizedConfiguration)
+        self.unauthorized_api_instance = conferences_api.ConferencesApi(unauthorized_api_client)            
+        
+        # Forbidden API Client
+
+        forbiddenConfiguration = bandwidth.Configuration(
+            username=FORBIDDEN_USERNAME,
+            password=FORBIDDEN_PASSWORD
+        )
+        forbidden_api_client = bandwidth.ApiClient(forbiddenConfiguration)
+        self.forbidden_api_instance = conferences_api.ConferencesApi(forbidden_api_client)
+
+        # Rest client for interacting with Manteca
+        self.rest_client = RESTClientObject(bandwidth.Configuration.get_default_copy())    
+        configuration = bandwidth.Configuration(
+            username = BW_USERNAME,
+            password = BW_PASSWORD,
+        )
+
+        self.account_id = BW_ACCOUNT_ID
+        self.callIdArray = []
+        self.testUpdateConf = UpdateConference(
+                     state=ConferenceStateEnum("active"),
+                     redirect_url=MANTECA_BASE_URL + "/bxml/pause",
+                     redirect_method=RedirectMethodEnum("POST"),
+                     username="mySecretUsername",
+                     password="mySecretPassword1!",
+                     redirect_fallback_url=MANTECA_BASE_URL + "/bxml/pause",
+                     redirect_fallback_method=RedirectMethodEnum("POST"),
+                     fallback_username="mySecretUsername",
+                     fallback_password="mySecretPassword1!",
+                     tag="My Custom Tag",
+                   )
+        self.testUpdateBxml = '<?xml version="1.0" encoding="UTF-8"?><Bxml><SpeakSentence locale="en_US" gender="female" voice="susan">This is test BXML.</SpeakSentence></Bxml>'
+        self.testUpdateMember = UpdateConferenceMember(mute=False)
+        self.testConfId = "Conf-id"
+        self.testMemberId = "Member-Id"
+        self.testRecordId = "Recording-Id"
+        self.TEST_SLEEP = 3
+        self.TEST_SLEEP_LONG = 10
+        
+    def tearDown(self):
+        callCleanup(self)
+
+    def assertApiException(self, context: ApiException, expectedException: ApiException, expected_status_code: int):
```

Comment:
We should probably move this to a test utils module or something since its reused in multiple tests

Reply:
Doesnt need to get done here - just thinking out loud